config/main

```diff
@@ -0,0 +1 @@
+include config/default
```

Comment:
Perhaps we should adapt the config inheritance in the UI and API when the
docker approach will be implemented (as it is already in v6_images?).

2016-06-30 16:34 GMT+02:00 Arnaud Morvan <notifications@github.com>:

> In config/main
> <https://github.com/c2corg/v6_images/pull/4#discussion_r69143351>:
>
> > @@ -0,0 +1 @@
> > +include config/default
>
> And we can commit dev / demo and prod configurations to github
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/c2corg/v6_images/pull/4/files/aef1b25cc12cb8f73f4e8b58a16f7ef235d5742d#r69143351>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABIxizLXFw-qYeZe1yaNl4dgDWPVy9dbks5qQ9PbgaJpZM4JAHsD>
> .
>